src/pages/RepoPage/CoverageOnboarding/CircleCI/CircleCI.tsx

```diff
@@ -38,110 +38,142 @@ function CircleCI() {
-          <CopyClipboard string={orbsString} />
-        </div>
-        <small>
-          For more, see Codecov specific{' '}
+      <Step1
+        tokenCopy={tokenCopy}
+        uploadToken={uploadToken}
+        providerName={providerName}
+      />
+      <Step2 defaultBranch={data?.repository?.defaultBranch ?? ''} />
+      <Step3 />
+      <FeedbackCTA />
+    </div>
+  )
+}
+
+export default CircleCI
+
+interface Step1Props {
+  tokenCopy: string
+  uploadToken: string
+  providerName: string
+}
+
+function Step1({ tokenCopy, uploadToken, providerName }: Step1Props) {
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title size="base">
+          Step 1: add {tokenCopy} token to{' '}
           <A
-            to={{ pageName: 'circleCIOrbs' }}
+            hook="circleCIEnvVarsLink"
             isExternal
-            hook="circleCIOrbsLink"
+            to={{
+              pageName: 'circleCIEnvVars',
+              options: { provider: providerName },
+            }}
           >
-            CircleCI Documentation
+            environment variables
           </A>
-        </small>
-      </div>
-      <ExampleBlurb />
-      <div>
+        </Card.Title>
+      </Card.Header>
+      <Card.Content className="flex flex-col gap-4">
         <p>
-          After you committed your changes and ran the repo&apos;s CI/CD
-          pipeline. In your pull request, you should see two status checks and
-          PR comment.
+          Environment variables in CircleCI can be found in project settings.
         </p>
-        <img
-          alt="codecov patch and project"
-          src={patchAndProject.toString()}
-          className="my-3 md:px-5"
-          loading="lazy"
-        />
+        <div className="flex gap-4">
+          <pre className="flex basis-1/3 items-center justify-between gap-2 rounded-md border border-ds-gray-secondary bg-ds-gray-primary p-4 font-mono">
+            <div className="w-0 flex-1 overflow-hidden" data-testid="token-key">
+              CODECOV_TOKEN
+            </div>
+            <CopyClipboard string="CODECOV_TOKEN" />
+          </pre>
+          <pre className="flex basis-2/3 items-center justify-between gap-2 rounded-md border border-ds-gray-secondary bg-ds-gray-primary p-4 font-mono">
+            <div className="w-0 flex-1 overflow-hidden">{uploadToken}</div>
+            <CopyClipboard string={uploadToken ?? ''} />
+          </pre>
+        </div>
+      </Card.Content>
+    </Card>
+  )
+}
+
+interface Step2Props {
+  defaultBranch: string
+}
+
+function Step2({ defaultBranch }: Step2Props) {
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title size="base">
+          Step 2: add Codecov orb to CircleCI{' '}
+          <A
+            hook="circleCIyamlLink"
+            isExternal
+            to={{
+              pageName: 'circleCIyaml',
+              options: { branch: defaultBranch },
+            }}
+          >
```

Comment:
What happens to this link of `defaultBranch` is an empty string? We may want this to actually possible be `undefined` so that the branch (param?) isn't passed along in the first place if we don't have any branch data to pass along

Reply:
Ahhh interesting ...